mage_ai/io/s3.py

```diff
@@ -124,21 +124,33 @@ def export(
                 )
 
     @classmethod
-    def with_config(cls, config: Mapping[str, Any]) -> 'S3':
-        try:
-            aws_config = config[IOConfigKeys.AWS]
-            s3_config = aws_config[IOConfigKeys.S3]
-        except KeyError:
-            raise KeyError(
-                f'No configuration settings found for '
-                f'\'{IOConfigKeys.AWS}.{IOConfigKeys.S3}\' under profile'
-            )
-        credentials = ['access_key_id', 'secret_access_key', 'region']
-        parameters = ['aws_access_key_id', 'aws_secret_access_key', 'region_name']
-        for credential, parameter in zip(credentials, parameters):
-            if credential in aws_config:
-                s3_config[parameter] = aws_config[credential]
-        return cls(**s3_config)
+    def with_config(
+        cls,
+        bucket_name: str,
```

Comment:
Did you test S3 data loader? The template uses `S3.with_config(ConfigFileLoader(config_path, config_profile)).load()` but the with_config method requires bucket_name and object_key.

Reply:
Okay yeah I tested the S3 client, but not with the template so I missed this. I'll fix.